notebooks/common/documentation/dataset-information.md

```diff
@@ -0,0 +1,13 @@
+### Loading profile data as Hatchet GraphFrame
+
+Hatchet queries are only defined on Hatchet GraphFrames. 
+Obtaining a hatchet GraphFrame is straight forward:
+
+1. Import hatchet
+2. Use the appropriate reader for the profile/trace at hand
+
+We first load a [Caliper](https://github.com/LLNL/Caliper) profile in JSON format and Caliper is a performance profiling library.
+
+This example profile is profiled from [LULESH (Livermore Unstructured Lagrangian Explicit Shock Hydrodynamics)](https://asc.llnl.gov/codes/proxy-apps/lulesh), a performance report data generated by Caliper. LULESH is a highly simplified application designed to solve the Sedov Blast problem, which is a standard hydrodynamics test problem. It performs a hydronomics stencil calculation using both MPI and OpenMP to achieve parallelism. 
```

Comment:
Should this be hydrodynamics stencil calculation?

Reply:
Yes, indeed. Thank you!